### PR TITLE
(MODULES-8771) - Release Prep 2.14.0

### PR DESCRIPTION
**Pull request by pmcmaw**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,13 +3,30 @@
...
+
+**Implemented enhancements:**
+
+- \(feat\) dont load the beaker if litmus is there [\#281](https://github.com/puppetlabs/puppetlabs_spec_helper/pull/281) ([tphoney](https://github.com/tphoney))
+- \(maint\) load rake tasks from optional libraries [\#279](https://github.com/puppetlabs/puppetlabs_spec_helper/pull/279) ([DavidS](https://github.com/DavidS))
+- Document how to set default values for fixture loading [\#277](https://github.com/puppetlabs/puppetlabs_spec_helper/pull/277) ([Felixoid](https://github.com/Felixoid))
+
+**Merged pull requests:**
+
+- Remove `--color` from everywhere, use RSpec default detection instead [\#280](https://github.com/puppetlabs/puppetlabs_spec_helper/pull/280) ([DavidS](https://github.com/DavidS))
```
Comment (tphoney) [inline CHANGELOG.md]: is this the new norm ?

Comment (DavidS) [reply CHANGELOG.md]: Yes, rspec automatically uses colors these days if running on a terminal. Also see #170 for background.

Comment (tphoney) [reply CHANGELOG.md]: sorry @DavidS it was about the "merged pull requests"

Comment (DavidS) [reply CHANGELOG.md]: Oh, meh. That heading is for non-labeled PRs. I've added "bug" to this PR, the changelog will get fixed next time it's rendered.

**Pull request merged**